Sources/GRPCInProcessTransport/InProcessTransport+Client.swift

```diff
@@ -297,11 +297,17 @@ extension InProcessTransport {
         }
       }
 
+      let clientContext = ClientContext(
+        descriptor: descriptor,
+        remotePeer: "<in-process>",
+        localPeer: "<in-process>"
+      )
```

Comment:
Sorry, forgot about this in the last review but the server adds the PID as well. We actually create the string for the server here: https://github.com/grpc/grpc-swift/blob/76073a2c1a2dcae35df4f0ab83ffd07b23e4cec9/Sources/GRPCInProcessTransport/InProcessTransport.swift#L28-L30 so we can just pass that in.